otx/cli/tools/train.py

```diff
@@ -207,20 +207,21 @@ def main():
         args.save_model_to = "./models"
     save_model_data(output_model, args.save_model_to)
 
-    validation_dataset = dataset.get_subset(Subset.VALIDATION)
-    predicted_validation_dataset = task.infer(
-        validation_dataset.with_empty_annotations(),
-        InferenceParameters(is_evaluation=False),
-    )
+    if data_config["data"]["val"]["data-roots"]:  # TODO (sungchul): refactoring
```

Comment:
Is todo resolved, or not? If resolved please remove comments.

Reply:
We decided to leave this part alone. I'll remove the comment.